### PR TITLE
Setup caikit-tgis-serving for RHODS 1.32

### PR DESCRIPTION
**Pull request by heyselbi**
- Setting up TGIS for RHODS 1.32
- Updated nightly to final tag
- added the files from running `make ci-operator-config jobs prow-config`

### CODE REVIEW

File: ci-operator/config/red-hat-data-services/caikit-tgis-serving/red-hat-data-services-caikit-tgis-serving-rhods-1.32.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: nightly-image-mirror
+  cron: 0 0 1 * * ?
```
Comment (Jooho) [inline ci-operator/config/red-hat-data-services/caikit-tgis-serving/red-hat-data-services-caikit-tgis-serving-rhods-1.32.yaml:28]: Still cron is here, please use postsubmit

**Comment by heyselbi** [comment]
/pj-rehearse pull-ci-red-hat-data-services-caikit-tgis-serving-rhods-1.32-images

**Comment by Jooho** [comment]
/pj-rehearse pull-ci-red-hat-data-services-caikit-tgis-serving-rhods-1.32-images

File: ci-operator/config/red-hat-data-services/caikit-tgis-serving/red-hat-data-services-caikit-tgis-serving-rhods-1.32.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    namespace: openshift
+    tag: golang-1.13
+images:
+- context_dir: .
+  dockerfile_path: Dockerfile
+  from: base
+  to: caikit-tgis-serving
+promotion:
+  namespace: red-hat-data-services
+  tag: rhods-1.32
```
Comment (Jooho) [inline ci-operator/config/red-hat-data-services/caikit-tgis-serving/red-hat-data-services-caikit-tgis-serving-rhods-1.32.yaml:18]: This should be matched with this :https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/text-generation-inference/red-hat-data-services-text-generation-inference-rhods-1.32.yaml#L17

You can update this line or `text-generation-inferenece` repo.

File: ci-operator/config/red-hat-data-services/caikit-tgis-serving/red-hat-data-services-caikit-tgis-serving-rhods-1.32.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
+base_images:
+  base:
+    name: text-generation-inference
+    namespace: red-hat-data-services
+    tag: rhods-1.32
```
Comment (Jooho) [inline ci-operator/config/red-hat-data-services/caikit-tgis-serving/red-hat-data-services-caikit-tgis-serving-rhods-1.32.yaml:5]: This should be matched with this :https://github.com/openshift/release/blob/master/ci-operator/config/red-hat-data-services/text-generation-inference/red-hat-data-services-text-generation-inference-rhods-1.32.yaml#L17

You can update this line or text-generation-inferenece repo.

**Comment by heyselbi** [comment]
/pj-rehearse pull-ci-red-hat-data-services-caikit-tgis-serving-rhods-1.32-images

**Pull request merged**